src/logcollector/read_win_event_channel.c

```diff
@@ -522,12 +523,22 @@ DWORD WINAPI event_channel_callback(EVT_SUBSCRIBE_NOTIFY_ACTION action, os_chann
 {
     if (action == EvtSubscribeActionDeliver) {
         send_channel_event(evt, channel);
+    } else {
+        while(1) {
```

Comment:
You cannot read other channels if `EventLog` is down. If you mean `logcollector` in general it will still work (I tried with a `localfile` and it'll work).